src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+/**
+ * Useful scraps of this and that.
+ */
+public final class SVUtils {
+
+    /**
+     * return a good initialCapacity for a HashMap that will hold a given number of elements
+     */
+    public static int hashMapCapacity( final int nElements )
+    {
+        return (int)((nElements*4L)/3) + 1;
+    }
+
+    /**
+     * Iterator over 1 or 0 elements.
+     */
+    public static final class SingletonIterator<T> implements Iterator<T> {
```

Comment:
Is there a need for these, instead of using `Collections.<T> emptyIterator()`, or `org.apache.commons.collections.iterators.SingletonIterator`?